apps/etterlatte-beregning/src/main/kotlin/ytelseMedGrunnlag/YtelseMedGrunnlagService.kt

```diff
@@ -59,8 +59,23 @@ class YtelseMedGrunnlagService(
 
         return YtelseMedGrunnlagDto(
             perioder = avkortinger,
+            inntektForNesteAar = harInntektForNesteAar(avkortingUtenLoependeYtelse),
         )
     }
+
+    private fun harInntektForNesteAar(avkortingUtenLoependeYtelse: Avkorting): Boolean {
+        if (avkortingUtenLoependeYtelse.aarsoppgjoer.isNotEmpty()) {
+            avkortingUtenLoependeYtelse.aarsoppgjoer.forEach { aarsoppgjoer ->
```

Comment:
Aarsoppgjør har et felt "aar" du kan bruke. Trenger ikke bruke noen av de underliggende periodene.
Og isåfall burde det blitt brukt AvkortingGrunnlag.

Reply:
Ja, hadde det som utgangspunkt, men stusset over at jeg kun fikk 1 årsoppgjør.. muligens test dataen min er feil